tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -993,27 +986,14 @@ public function test_wp_set_script_translations_i18n_dependency() {
 	}
 
 	/**
-	 * @ticket 45103
+	 * @ticket 55250
```

Comment:
Is it possible to keep both?

Reply:
Yes, pushed a commit that only adds a second `@ticket` reference and doesn't remove the first one.